src/index.js

```diff
@@ -3,16 +3,21 @@ import PropTypes from 'prop-types';
 
```

Comment:
Just a suggestion

`handleMessageAddition` function can be refactored like below
```
messages.push(
        <Message
          message={isOnline ? onlineMessage : offlineMessage}
          style={{
            ...messageStyles,
            backgroundColor: isOnline ? onlineColor : offlineColor,
          }}
          className={messageClassName}
        />,
      );
```

Reply:
Thanks I will do that!